syntax = "proto3";

import "alice/wonderlogs/library/protos/banned_users.proto";

import "library/cpp/getoptpb/proto/confoption.proto";

message TConfig {
    option (NGetoptPb.Title) = "Canonizer";

    message TYt {
        string Cluster = 1;
        string Pool = 2;
        string WonderDir = 3;
        string RawLogsDir = 4;
    }

    TYt Yt = 1;
    string Date = 2;
    int64 SplitServerTimeMs = 3;
    string OutputDir = 4;
    int64 StableSandboxResourceId = 5;
    TBannedUsers BannedUsers = 6;
}
